taskcluster/mozillavpn_taskgraph/transforms/signing.py

```diff
@@ -20,6 +20,7 @@
     "linux/opt",
     "macos/opt",
     "windows/opt",
+    "addons/opt",
```

Comment:
Note that until the `release` signer exists in autograph, production signing tasks will fail. To avoid confusion, maybe we should add it to the `SIGNING_BUILD_TYPES` below so we don't start running this in L3.

I believe this will also start running the manifest signing task on every push, which we may want to avoid as I don't see much benefit in running them outside the relpro action.

Reply:
OK. I'll try it out in staging soon

Reply:
Still TODO